datalad/distributed/ora_remote.py

```diff
@@ -883,6 +874,39 @@ def _verify_config(self, gitdir, fail_noid=True):
             self.storage_host, self.store_base_path, self.ria_store_url = \
                 verify_ria_url(self.ria_store_url, url_cfgs)
 
+        else:
+            # for now still accept the configs, if no ria-URL is known, but
+            # issue deprecation warning:
+            host = _get_gitcfg(gitdir,
+                               'annex.ora-remote.{}.ssh-host'.format(name)) or \
+                   self.annex.getconfig('ssh-host')
+            self.storage_host = None if host == '0' else host
+
+            path = _get_gitcfg(gitdir,
+                               'annex.ora-remote.{}.base-path'.format(name)) or \
+                   self.annex.getconfig('base-path')
+            self.store_base_path = path.strip() if path else path
+
+            if path or host:
+                self.message("WARNING: base-path + ssh-host configs are "
+                             "deprecated and won't be considered in the future."
```

Comment:
Missing a space between "future." and "Use".

Reply:
Thx.